embulk-output-redshift/build.gradle

```diff
@@ -31,3 +31,62 @@ embulkPlugin {
     category = "output"
     type = "redshift"
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Hitoshi Tanaka"
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "Hiroyuki Sato"
+                        email = "hiroysato@gmail.com"
+                    }
+                    // developer {  // https://github.com/embulk/embulk-output-jdbc/commit/97f3a4ba14ecb99b9bdfa03831a84cb7e1a8e970
+                    //     name = "Lance Norskog"
+                    //     email = "lance@edmodo.com"
+                    // }
```

Comment:
@hito4t Do you remember this merge commit 97f3a4ba14ecb99b9bdfa03831a84cb7e1a8e970 ?

It looks to be commits from Lance Norskog, but not via a pull request.

Reply:
@dmikurube 
I was not able to merge the PR #86 as it was, so I modified and merged.

Reply:
Thanks! Got it. And the person is no longer on GitHub... Let's skip them.